.github/dependabot.yml

```diff
@@ -6,6 +6,6 @@ updates:
     schedule:
       interval: daily
     reviewers:
-      - markdorison
+      - apotek
     assignees:
-      - markdorison
+      - apotek
```

Comment:
This is a great and much-needed update! It makes me wonder whether we could name a _team_ instead of an individual. Like, say, our @chromatichq/devops team, which is a thing that exists and which currently includes @apotek, @mattdaniel, and @mmatsoo. Thoughts?

Reply:
Do you want me to try it?

Reply:
Ok. I tried it. If it fails, no biggie. We'll see what happens.